fixups/RegLegacyFixups/RegistryFixups.cpp

```diff
@@ -257,6 +261,28 @@ REGSAM RegFixupSam(std::string keypath, REGSAM samDesired, DWORD RegLocalInstanc
     return samModified;
 }
 
+template <typename CharT>
+const char* ReplaceRegistryQueryPath(PHKEY key, const CharT* subKey)
+{
+    std::string keypath = InterpretKeyPath(*key) + "\\" + InterpretStringA(subKey);
```

Comment:
crash if `subkey` is null

Reply:
How? https://github.com/microsoft/MSIX-PackageSupportFramework/blob/81180e351e600afab9eb3ffb80661d36322b6f77/fixups/RegLegacyFixups/Logging.h#L250

Reply:
Interesting. I would've looked at `IntepretStringA`  Just because code now checks for a null pointer, does not mean it'll change and subkey is used without checking if `subKey` is null.  

Please add a comment mentioning why no null check.  Something like "No null check is made because the only place subkey is used is IntepretStringA and they handles nullptr"

Reply:
Added _In_opt_ annotation explicitly denote that `subKey` can be null